src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random = new Random();
+
+    public Color getRandomColor() {
+        int index = random.nextInt(Color.values().length);
+        return Color.valueOf(Color.values()[index].name());
```

Comment:
```suggestion
        return Color.valueOf(Color.values()[index].name());
```
Превратить энум в стринг чтобы потом этот стринг превратить в энум? В чем смысл этих преобразований?)))